flepimop/gempyor_pkg/src/gempyor/parameters.py

```diff
@@ -123,30 +123,27 @@ def __init__(
                     print("loaded col :", sorted(list(df.columns)))
                     print("geodata col:", sorted(subpop_names))
                     raise ValueError(
-                        f"""ERROR loading file {fn_name} for parameter {pn}: the number of non 'date'
-                    columns are {len(df.columns)}, expected {len(subpop_names)} (the number of subpops) or one."""
+                        f"Issue loading file '{fn_name}' for parameter '{pn}': "
+                        f"the number of non-'date' columns is '{len(df.columns)}', "
+                        f"expected '{len(subpop_names)}' (number of subpopulations) or one."
                     )
 
                 df = df[str(ti) : str(tf)]
                 if not (len(df.index) == len(pd.date_range(ti, tf))):
                     print("config dates:", pd.date_range(ti, tf))
                     print("loaded dates:", df.index)
                     raise ValueError(
-                        f"""ERROR loading file {fn_name} for parameter {pn}: 
-                    the 'date' entries of the provided file do not include all the days specified to be modeled by 
-                    the config. the provided file includes {len(df.index)} days between {str(df.index[0])} to {str(df.index[-1])}, 
-                    while there are {len(pd.date_range(ti, tf))} days in the config time span of {ti}->{tf}. The file must contain entries for the
-                    the exact start and end dates from the config. """
+                        f"Issue loading file '{fn_name}' for parameter '{pn}': " 
+                        f"Provided file dates span '{str(df.index[0])}' to '{str(df.index[-1])}', "
+                        f"but the config dates span '{ti}' to '{tf}'."
                     )
                 if not (pd.date_range(ti, tf) == df.index).all():
                     print("config dates:", pd.date_range(ti, tf))
                     print("loaded dates:", df.index)
                     raise ValueError(
-                        f"""ERROR loading file {fn_name} for parameter {pn}: 
-                    the 'date' entries of the provided file do not include all the days specified to be modeled by 
-                    the config. the provided file includes {len(df.index)} days between {str(df.index[0])} to {str(df.index[-1])}, 
-                    while there are {len(pd.date_range(ti, tf))} days in the config time span of {ti}->{tf}. The file must contain entries for the
-                    the exact start and end dates from the config. """
+                        f"Issue loading file '{fn_name}' for parameter '{pn}': "
+                        f"Provided file dates span '{str(df.index[0])}' to '{str(df.index[-1])}', "
+                        f"but the config dates span '{ti}' to '{tf}'."
```

Comment:
same as above

Reply:
opting to keep dates/all info in the error message, especially if you think it's helpful. i can alter it later if you choose!